### DISCUSSION TITLE
config file content completion

### DISCUSSION THREAD

**Discussion by edisonX-sudo** [lowdefy/lowdefy#1147] [Questions and Help]
Hi @Gervwyk 
will it be possible to make YAML/JSON support Feature like content completion like XML does?
or just make lowdefy support XML to gain the feature.
I`m sure YAML is quite a readable format 4 config file but it's killing me 2 complete config files without features like content completion 😭

**Comment by Gervwyk** [comment]
Hi @edisonX-sudo

Yeah. It is possible, and we'd really like to add it once we have some additional resources. We've unpacked it a little in this thread https://github.com/lowdefy/lowdefy/discussions/677

In the meantime, it would be great if someone from the community is looking for a challenge 😎
